### PR TITLE
Refactoring of opened editors list sample plugin

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Refactoring of Opened Editors List sample plugin
- rename sample plugin to opened-editors-list
- refactor the plugin
- commenting the code
- display only fie name instead full path
- highlight the tree item when change the active editor

opened-editors-list-12

Signed-off-by: Vitaliy Gulyy <vgulyy@redhat.com>

### CODE REVIEW

File: samples/opened-editors-list/package.json [hunk 1]
```diff
@@ -1,16 +1,15 @@
 {
-      "name": "opened-editor-list",
-      "publisher": "theia",
-      "keywords": [
+    "name": "opened-editors-list",
+    "publisher": "theia",
```
Comment (benoitf) [inline samples/opened-editors-list/package.json]: eclipse-che ?

File: samples/opened-editors-list/src/opened-editors-list-frontend.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import * as theia from '@theia/plugin';
```
Comment (benoitf) [inline samples/opened-editors-list/src/opened-editors-list-frontend.ts]: copyright headers

File: samples/opened-editors-list/package.json [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
+    "keywords": [
         "theia-plugin"
-      ],
-      "version": "0.0.1",
-      "license": "none",
-
-      "files": [
+    ],
+    "version": "0.0.1",
+    "license": "none",
```
Comment (benoitf) [inline samples/opened-editors-list/package.json]: EPL2.0

File: samples/opened-editors-list/package.json (hunk 1 above)
Comment (vitaliy-guliy) [reply samples/opened-editors-list/package.json]: Do you mean `@eclipse-che/opened-editors-list` ??

If yes, we need to separate samples from real plugins stored in https://github.com/eclipse/che-theia/tree/master/plugins

`@eclipse-che/samples/opened-editors-list` ?

Comment (AndrienkoAleksandr) [reply samples/opened-editors-list/package.json]: > @eclipse-che/samples/opened-editors-list ?

I think no - https://docs.npmjs.com/about-scopes
`@eclipse-che` - it's scope of the package, `opened-editors-list` - node package name.

Comment (benoitf) [reply samples/opened-editors-list/package.json]: I was referencing the "publisher" : eclipse-che instead of theia

Comment (vitaliy-guliy) [reply samples/opened-editors-list/package.json]: Done

File: samples/opened-editors-list/package.json (hunk 3 above)
Comment (vitaliy-guliy) [reply samples/opened-editors-list/package.json]: Done

**Comment by vitaliy-guliy** [comment]
We have to review all the samples: add copyright headers and add license to package.json

**Pull request merged**